BRPeerManager.c

```diff
@@ -739,6 +740,98 @@ static void _BRPeerManagerFindPeers(BRPeerManager *manager)
     }
 }
 
+// DNS peer discovery V2 (New)
+static void _BRPeerManagerFindPeersV2(BRPeerManager *manager)
+{
+    uint64_t services = SERVICES_NODE_NETWORK | SERVICES_NODE_BLOOM | manager->params->services;
+    time_t now = time(NULL);
+    struct timespec ts;
+    pthread_t thread;
+    pthread_attr_t attr;
+    UInt128 *addr, *addrList;
+    BRFindPeersInfo *info;
+
+    //TODO: WIP HERE, get from shared prefs
+    // List of hardcoded IP addresses (replace with actual IPs for your network)
+    // Ensure these peers are reliable and support SPV mode.
+    const char *hardcodedIPs[] = {
```

Comment:
does it something like this? I mean the IPs will obtained from [the API](https://api.blockchair.com/litecoin/nodes) ?

Reply:
Yes @andhikayuana .  We are working with the backend to deliver the filtered nodes where the mobile-api can give an array and it will have the freshest list.
For the short term, DM as I can show you some manually created lists